### PR TITLE
7916 - Ensure isSealed is indexed and returned in case search

### PR DESCRIPTION
**Pull request by matthopson**
Note this will require an ES re-index.

### CODE REVIEW

File: shared/src/persistence/elasticsearch/caseAdvancedSearch.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ exports.caseAdvancedSearch = async ({ applicationContext, searchTerms }) => {
     'docketNumberSuffix',
     'docketNumberWithSuffix',
     'irsPractitioners',
+    'isSealed',
```
Comment (mrinsin) [inline shared/src/persistence/elasticsearch/caseAdvancedSearch.js:30]: Nit: is there a unit test we could add to test this addition?

**Review by mrinsin — APPROVED** [review]

File: shared/src/persistence/elasticsearch/caseAdvancedSearch.js (hunk 1 above)
Comment (matthopson) [reply shared/src/persistence/elasticsearch/caseAdvancedSearch.js:30]: Yeah it's tough. We can't test that the mapping looks right or that ES is indexing the right fields... I wanted to step away from it yesterday and come back and think about that a little this morning before merging. I should probably put a label on it.

**Review by rachaelparris — COMMENTED** [review]
Is this to fix a bug or for a trello card? Could you add an issue number?

**Review by rachaelparris — COMMENTED** [review]
Why is `sealedDate` not filtering out the case? Is there no `sealedDate` on the case? Should we be _replacing_ `sealedDate` with `isSealed` instead if it's not working correctly?

**Comment by matthopson** [comment]
@rachaelparris There is no sealedDate on some cases - I assume those are migrated cases where it had a sealed flag but no date. If you look at the `isSealedCase` method on `Case`, it evaluates both for this purpose. So the system expects this - we even pass it into the caseFilter's list of fields to evaluate... it's just not getting returned from ES anymore (it was removed).

**Pull request merged**